packages/suite/src/components/suite/CoinsGroup/CoinsList.tsx

```diff
@@ -65,19 +28,19 @@ const CoinsList = ({ onToggleFn, networks, selectedNetworks, unavailableCapabili
         <Wrapper>
             {networks.map(({ symbol, accountType, label, tooltip, name }) => {
                 const isSelected = selectedNetworks.includes(symbol);
-                const isDisabled = !!unavailableCapabilities?.[symbol] || isDeviceLocked;
-                const tooltipContent = getTooltipContent({
-                    tooltip,
-                    device,
-                    isDisabled,
-                    whyDisabled: unavailableCapabilities?.[symbol],
-                });
+                const isUnavailable = device.unavailableCapabilities?.[symbol];
```

Comment:
nit: `isUnavailable` sounds like a boolean (because of the `is` at the beginning) but it's type of `UnavailableCapability | undefined`. 
```suggestion
                const isUnavailable = !!device.unavailableCapabilities?.[symbol];
```

Reply:
58dcf8bbf3612d92328da5557bd5e55c62c500f7